src/NexyCrypt.php

```diff
@@ -299,7 +320,7 @@ private function signedPostRequest($uri, array $payload)
      *
      * @return ResponseInterface
      */
-    private function request($method, $uri, array $options = [])
+    private function request($method, $uri, array $options = ['verify' => false])
```

Comment:
Hi @Soullivaneuh , if we remove the verify key, it will get the message: ```curl: (60) SSL certificate problem: Invalid certificate chain```. I think the proper way is pass all SSL certificate and that's why I add this key.